duckduckgo/macos-browser
# Partial tech debt cleanup

Task/Issue URL:
* https://app.asana.com/0/1199230911884351/1201394179028190/f
* https://app.asana.com/0/1199230911884351/1201394179028195/f 

CC: @tomasstrba 

**Description**:

This PR cleans up the tech debt items raised in these tasks: [one](https://app.asana.com/0/1199230911884351/1201394179028195/f), [two](https://app.asana.com/0/1199230911884351/1201394179028190/f).

**Steps to test this PR**:
1. Run the app and verify that the logging specified in the tasks is gone
1. Test that the suggestions table view works as expected

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
